### PR TITLE
Edited the mecanum drive example code to be more frc-docs like.

### PR DESCRIPTION
**Pull request by BerkeSinanYetkin**
As stated in the #49 issue in the base repository, we need example code that matches WPIlib examples closely.

I may have broken some PEP guidelines, but I believe we need formatting to be in line with C++/Java examples.

The code has been tested in simulation only, because my team does not have any SparkMax motor controllers.

### CODE REVIEW

File: mecanum-drive/robot.py [hunk 1]
```diff
@@ -6,52 +6,38 @@
...
-        self.drive.setExpiration(0.1)
+        self.frontRight.setInverted(True)
+        self.rearRight.setInverted(True)
 
-        self.stick = wpilib.Joystick(self.joystickChannel)
+        self.m_drive = MecanumDrive(self.frontLeft, self.rearLeft, self.frontRight, self.rearRight)
 
-    def teleopInit(self):
-        self.drive.setSafetyEnabled(True)
+        self.m_stick = Joystick(self.kJoystickChannel)
```
Comment (auscompgeek) [inline mecanum-drive/robot.py:35]: Our examples drop the `m_` prefix as it's already clear that these are member variables by the use of `self`. We settled on this in the frc-docs PR too.

File: mecanum-drive/robot.py [hunk 2]
```diff
@@ -6,52 +6,38 @@
 
 import wpilib
 from wpilib.drive import MecanumDrive
+from wpilib import Joystick, PWMSparkMax, TimedRobot
```
Comment (auscompgeek) [inline mecanum-drive/robot.py:9]: I believe [we standardised on avoiding `from wpilib import` over in frc-docs][frc-docs #1807] (cf. [the Zen of Python][PEP 20]). For anything that's in the root `wpilib` namespace, we should just reference `wpilib.` similarly to how the C++ examples don't do `using namespace frc;` or similar.

[frc-docs #1807]: https://github.com/wpilibsuite/frc-docs/pull/1807
[PEP 20]: https://peps.python.org/pep-0020/

**Review by auscompgeek — COMMENTED** [review]
Most of the renames, including the constant prefix `k`, look reasonable to me. I'd like a couple of changes undone so that this actually is more frc-docs like though.

You'll need to run the `black` autoformatter after making those changes.

File: mecanum-drive/robot.py (hunk 1 above)
Comment (BerkeSinanYetkin) [reply mecanum-drive/robot.py:35]: on it.

File: mecanum-drive/robot.py (hunk 2 above)
Comment (BerkeSinanYetkin) [reply mecanum-drive/robot.py:9]: on it.

**Review by auscompgeek — APPROVED** [review]
Thank you for helping keep our examples up-to-date :sparkles:

**Pull request merged**